index.js

```diff
@@ -5,6 +5,7 @@ const cors = require("cors");
 const dotenv = require("dotenv");
 dotenv.config();
 const getAllWarehouses = require("./routes/getallwarehouses/getAllWarehouses");
```

Comment:
should be camelcase for 'getallwarehouses' here?

Reply:
that's caitlin's I will let her know